hnn_core/network.py

```diff
@@ -373,13 +379,13 @@ def __init__(self, params, add_drives_from_params=False,
         # cell counts, real and artificial
         self.pos_dict = dict()
         self.cell_types = dict()
-        pos = _create_cell_coords(n_pyr_x=self._params['N_pyr_x'],
-                                  n_pyr_y=self._params['N_pyr_y'],
-                                  zdiff=1307.4)
-        self.pos_dict['origin'] = pos['origin']
+
+        self._N_pyr_x = self._params['N_pyr_x']
+        self._N_pyr_y = self._params['N_pyr_y']
```

Comment:
Is there a particular reason why these are attributes of `Network`?

Reply:
also, not sure if it's updated after calling `set_cell_positions` ...

Reply:
Good questions, which revealed a bug (calling `set_cell_positions` would inadvertently set all non-specified parameters to their default values). This is fixed in ff7fb202